plugins/circleci/src/components/BuildWithStepsPage/BuildWithStepsPage.tsx

```diff
@@ -35,14 +29,12 @@ import {
   Link,
 } from '@backstage/core-components';
 
-const IconLink = IconButton as any as typeof MaterialLink;
-
 const BuildName = ({ build }: { build?: BuildWithSteps }) => (
   <Box display="flex" alignItems="center">
     #{build?.build_num} - {build?.subject}
-    <IconLink href={build?.build_url} target="_blank">
+    <IconButton href={build?.build_url || '#'} target="_blank">
       <LaunchIcon />
-    </IconLink>
+    </IconButton>
```

Comment:
So a possible recommendation, just to keep the components consistent through the Backstage UI?

I tested this locally and it works:

```typescript
    <Button to={build?.build_url || '#'} target="_blank">
      <LaunchIcon />
    </Button>
```

this eliminates needing to import Material UI's IconButton component, and also when adding Backstage's component, the spacing/formatting SLIGHTLY changes which also means it's common across multiple plugins.

### Current Master

<img width="690" alt="image" src="https://user-images.githubusercontent.com/33203301/196703812-c3f9183b-cffa-4707-a037-61864e100960.png">

(notice the hover target is circular)

### Your PR

Perfectly changes a component without any visual change to the UI.

<img width="690" alt="image" src="https://user-images.githubusercontent.com/33203301/196703512-50f69c15-d838-445c-82aa-e5f8204ad25f.png">

(notice the hover target is circular)

### My Suggestion

<img width="690" alt="image" src="https://user-images.githubusercontent.com/33203301/196704197-98bf839e-1cdd-4284-9d07-86726ee1fd13.png">

(notice the hover target is now rectangular)

welcome input from the core maintainers also, but I just think aligning to the core components for UI consistency is super helpful for end users as they cross tabs in a component and cross into different plugins! :)

NB: you'd have to change imports at top of course, also.

Reply:
@adamdmharvey, thanks for looking into it!

> Your PR
> Perfectly changes a component without any visual change to the UI.

The naming is confusing 😄  The component is the same as before.

I simply removed `const IconLink = IconButton` part:
```diff
- import { IconButton, Link as MaterialLink} from '@material-ui/core';
+ import { IconButton } from '@material-ui/core';

- const IconLink = IconButton as any as typeof MaterialLink;
-
  const BuildName = (props) => (
-   <IconLink>
+   <IconButton>
      <LaunchIcon />
-   </IconLink>
+   </IconButton>
  )
```

---
Regarding the suggestion,

Sure, we can replace `IconButton` with `Button`.

May I ask, what's the issue with `IconButton`? 
I saw a few cases with `IconButton` in the codebase and assumed it was okay to use it.

Reply:
How about
```js
import { Link } from '@backstage/core-components';

<Link to={build?.build_url || '#'} target="_blank">
  <IconButton component="div">
    <LaunchIcon />
  </IconButton>
</Link>
```
?

Reply:
yeah i think i twas just recommending avoiding importing Material components directly, preferring to  import backstage-first person components!  :) (so the customizations become consistent across plugins)